install_config/install/prerequisites.adoc

```diff
@@ -806,6 +806,51 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+Sometimes a container's log file
+(*_/var/lib/docker/containers/HASH/HASH-json.log_*) can increase to a
+problematic size. You can manage this by configuring the `json-file` logging
+driver to restrict the size and number of log files.
+
+For example, `--log-opt max-size` rolls over the size, and you can specify the
```

Comment:
I'd suggest rewriting this so that the sentence structure is more uniform when explaining each option.